<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeStatusWidget</class>
 <widget class="QWidget" name="CustomizeStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="statusGridLayout" columnminimumwidth="0,150,0">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QLabel" name="slackEmojiLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Slack Emoji</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout3">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="colorWidget3" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton3">
        <property name="text">
         <string>Select a Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="colorWidget2" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton2">
        <property name="text">
         <string>Select a Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="colorWidget1" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton1">
        <property name="text">
         <string>Select a Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="nameLineEdit1"/>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="nameLineEdit3"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="slackEmojiLineEdit3">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="slackEmojiLineEdit2">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="colorLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="slackEmojiLineEdit1">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="nameLineEdit2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>ex: Available</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>ex: :smile:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
